style.css

```diff
@@ -396,17 +600,616 @@ ul {
+    width: 100%;
+    margin-right: 20px;
+  }
+
+  #toolbar nav ul.navbar {
+    display: flex;
+    justify-content: space-evenly;
+  }
+
+  #toolbar nav ul.active li {
+    display: flex;
+    padding: 0 20px;
+    font-weight: 600;
+  }
+
+  #about-myself .about-2 .programming-lang li {
+    width: 122px;
+    display: flex;
+    align-items: flex-start;
+    flex-direction: column;
+  }
+
+  #about-myself .about-1 ul.social li {
+    padding: 10px 10px;
+    margin: 10px 0;
+  }
+
+  #toolbar nav ul.navbar li:nth-of-type(2n),
+  #toolbar nav ul.navbar li:nth-of-type(3) {
+    padding-left: 30px;
+    font-weight: 500;
+  }
+
+  #toolbar nav ul.navbar li a {
+    color: var(--color3);
+  }
+
+  #headline {
+    background-color: var(--white);
+    background-image: url(./Icons/desktop/Header\ bg.svg);
+    background-repeat: no-repeat;
+    background-size: contain;
+    background-position: left top;
+    border-radius: 0 0 0 80px;
+    padding: 400px 0;
+  }
+
+  #headline .heading-section {
+    padding: 289px 260px;
+  }
+
+  #headline h1 {
+    width: 450px;
+    color: var(--color2);
+    font-size: 48px;
+    font-family: "Poppins", sans-serif;
+    font-weight: 700;
+    line-height: 60px;
+    letter-spacing: 0.37px;
+  }
+
+  #headline p {
+    font-size: 18px;
+    font-family: 'Poppins', sans-serif;
+    font-weight: 400;
+    font-style: normal;
+    line-height: 28px;
+    margin: 15px 0;
+  }
+
+  /* card  */
+
+  .card-works .card {
+    width: 90%;
+    border: 1px solid var(--color7);
+    border-radius: 24px;
+    display: grid;
+    justify-content: space-between;
+    grid-template-columns: repeat(2, 2fr);
+  }
+
+  .card-works .card:nth-child(even) {
+    direction: rtl;
+  }
+
+  .card-works .card:nth-child(odd) {
+    direction: ltr;
+  }
+
+  .card-works .card .img {
+    width: 100%;
+    height: 448px;
+    background-image: url('./Icons/desktop/1 Snapshoot Portfolio.png');
+    background-repeat: no-repeat;
+    background-size: cover;
+    border-radius: 10px;
+  }
+
+  .card-works .card .img-1 {
+    width: 47%;
+    height: 448px;
+    background-image: url('./Icons/desktop/2 Snapshoot Portfolio.png');
+    background-repeat: no-repeat;
+    background-size: cover;
+    border-radius: 10px;
+    order: 2;
+  }
+
+  .card-works .card .img-2 {
+    width: 47%;
+    height: 448px;
+    background-image: url('./Icons/desktop/3 Snapshoot Portfolio.png');
+    background-repeat: no-repeat;
+    background-size: cover;
+    border-radius: 10px;
+  }
+
+  .card-works .card .img-3 {
+    width: 47%;
+    height: 448px;
+    background-image: url('./Icons/desktop/4 Snapshoot Portfolio.png');
+    background-repeat: no-repeat;
+    background-size: cover;
+    border-radius: 10px;
+    order: 2;
+  }
+
+  .card-works .card .card-description,
+  .card-works .card .card-description-1,
+  .card-works .card .card-description-2,
+  .card-works .card .card-description-3 {
+    /* width: 100%; */
```

Comment:
- [ ] It's a good thing you currently load this part of your portfolio dynamically. How about removing the whole of this completely? Commenting old pieces of code in a workspace makes the code looks kinda messy :hammer_and_wrench: